rerun_py/rerun_sdk/rerun/archetypes/video_frame_reference_ext.py

```diff
@@ -67,3 +71,60 @@ def __init__(
             return
 
         self.__attrs_clear__()
+
+    @classmethod
+    def columns_seconds(
```

Comment:
Surprising that this one is called `columns_seconds` while the other two are called `milliseconds`/`nanoseconds`. Any reason for that?

Reply:
oversight -- i meant for all of them to be prefixed with `columns_`